modules/logging-6x-v-5x.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="logging-6x-v-5x_{context}"]
+= OpenShift Logging 6.0: Key Differences from Previous Versions
+
+OpenShift Logging 6.0 is the culmination of efforts to modernize the underlying components of the logging solution. Over the past few releases, Loki and Vector were gradually introduced with a focus on improving performance and enhancing features while maintaining the stability required for business continuity.
+
+
+* **Horizontal Scalability:** Loki's architecture allows for seamless horizontal scaling, ensuring that the logging system can handle the growing demands of large-scale OpenShift deployments.
+
+* **Improved Query Performance:** Loki's indexing and query mechanisms are optimized for speed, enabling faster log retrieval and analysis.
+
+* **Streamlined Configuration:** The removal of Elasticsearch and Fluentd simplify setup and configuration.
```

Comment:
It's arguable if this streamlines configuration.  These are just technologies we no longer offer in 6.0

Reply:
Simplifies? Going from 4 different config options to 1 is certainly...something.